SR2023/2022-10-17-place-rejection.md

```diff
@@ -0,0 +1,17 @@
+---
+to: Student Robotics 2023 rejections
+subject: Student Robotics 2023 is sadly full
+---
+
+Hello!
+
+I'm afraid Student Robotics 2023 is full, and we are unable to offer you a place. We've had more schools interested than we have places for.
```

Comment:
Do we want to explain why we're limited to the number we are? (i.e: kits with custom boards, space in the venue).
I think it's fine not to, just enumerating the things it might make sense to include.